influxql/ast.go

```diff
@@ -2822,6 +2822,10 @@ func Walk(v Visitor, node Node) {
 			Walk(v, c)
 		}
 
+	case *DropSeriesStatement:
```

Comment:
While we're in here, should we add a panic for unsupported cases? Or can this be called for unsupported cases during normal operation?